### PR TITLE
Add Plutus

### PR DESCRIPTION
**Pull request by butterdawgs**
Support for Plutus.it

### CODE REVIEW

**Comment by phallobst** [comment]
You could use this SVG version of the logo:
plutus it

File: entries/p/plutus.it.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+   "Site Name":{
```
Comment (Carlgo11) [inline entries/p/plutus.it.json]: Please replace "Site Name" with the site's name.

File: entries/p/plutus.it.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      "domain":"plutus.it",
+      "img":"plutus.it.png",
+      "tfa":[
+         "totp"
+      ],
+      "documentation":"https://support.plutus.it/hc/en-us/articles/7203346943517-Account-creation-password-reset-2FA-and-account-block-suspension",
```
Comment (Carlgo11) [inline entries/p/plutus.it.json]:
```suggestion
      "documentation":"https://support.plutus.it/hc/en-us/articles/7203346943517",
```

File: entries/p/plutus.it.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      "categories":[
+         "banking",
+         "finance"
```
Comment (Carlgo11) [inline entries/p/plutus.it.json]: I think these categories are a better fit for the service.
```suggestion
         "cryptocurrencies",
         "banking"
```

File: entries/p/plutus.it.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      ]
```
Comment (Carlgo11) [inline entries/p/plutus.it.json]: According to [this article](https://support.plutus.it/hc/en-us/articles/4702798801949), only customers from the following countries are accepted.
```suggestion
      ],
      "regions": [
         "at",
         "be",
         "bg",
         "hr",
         "cz",
         "dk",
         "fi",
         "fr",
         "de",
         "gr",
         "hu",
         "is",
         "ie",
         "it",
         "lv",
         "li",
         "lt",
         "lu",
         "mt",
         "nl",
         "pl",
         "pt",
         "cy",
         "ro",
         "sk",
         "si",
         "es",
         "se"
      ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/p/plutus.it.json (hunk 4 above)
Comment (butterdawgs) [reply entries/p/plutus.it.json]: You're missing UK and more.

 

United Kingdom

England
Northern Ireland
Scotland
Wales
 

European Economic Area

Austria
Belgium
Bulgaria
Croatia
Czech Republic
Denmark
Finland
France
Germany
Greece
Hungary
Iceland
Ireland
Italy
Latvia
Liechtenstein
Lithuania
Luxembourg
Malta
Netherlands
Poland
Portugal
Republic of Cyprus
Romania
Slovakia
Slovenia
Spain
Sweden

File: entries/p/plutus.it.json (hunk 3 above)
Comment (butterdawgs) [reply entries/p/plutus.it.json]: Agreed

**Comment by butterdawgs** [comment]
I am new to GitHub so I have no clue on how to edit it but I am looking into it.

**Review by butterdawgs — COMMENTED** [review]
All should be good now :)

Edit: The delete button is greyed out, I cannot delete plutus.it.png

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
I removed the old PNG image and pushed it to your fork.
Your contribution will be visible in around 10 minutes. Sometimes, it takes longer due to server and browser cache.

**Comment by butterdawgs** [comment]
Appreciate it. 

Also it appears Plutus is listed twice? 

I’m sorry if I messed anything up I’m new to all this

**Comment by Carlgo11** [comment]
It's listed in the banking and cryptocurrency categories, so everything is as it should be.

**Pull request merged**